crates/sui-node/src/lib.rs

```diff
@@ -212,9 +213,15 @@ impl SuiNode {
         ));
 
         let perpetual_options = default_db_options().optimize_db_for_write_throughput(4);
-        let store = AuthorityStore::open(
+        let perpetual_tables = Arc::new(AuthorityPerpetualTables::open(
             &config.db_path().join("store"),
             Some(perpetual_options.options),
+        ));
+        let empty_database = perpetual_tables
```

Comment:
Maybe leave a comment here saying that empty database indicates genesis

Reply:
Or maybe just rename the variable to `is_genesis`?

Reply:
done